### PR TITLE
fix: fix versions for indexers and rankers

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: indexers/vector/FaissIndexer/requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-faiss-cpu
+faiss-cpu==1.6.3
```
Comment (florian-hoenicke) [inline indexers/vector/FaissIndexer/requirements.txt:1]: does `faiss-cpu==1.6.5` work as well?

Comment (JoanFM) [reply indexers/vector/FaissIndexer/requirements.txt:1]: I do not know, I checked the current version and fixed the version that is currently installed, I can't go further and innovate fromt hat

Comment (florian-hoenicke) [reply indexers/vector/FaissIndexer/requirements.txt:1]: Pycharm was suggesting me `1.6.5` and now I tried it out. All tests pass.

Comment (JoanFM) [reply indexers/vector/FaissIndexer/requirements.txt]: Done!

**Review by florian-hoenicke — APPROVED** [review]

**Pull request merged**